src/Npgsql/NpgsqlCommand.cs

```diff
@@ -596,7 +596,7 @@ Task Prepare(bool async, CancellationToken cancellationToken = default)
             var needToPrepare = false;
             foreach (var statement in _statements)
             {
-                if (statement.IsPrepared)
+                if (statement.IsPrepared(out _))
```

Comment:
Could keep a parameterless IsPrepared property to avoid the `(out _)` all the time

Reply:
Na-ah, doesn't compile (CS0102: The type 'NpgsqlStatement' already contains a definition for 'IsPrepared').

Reply:
Then IsPrepared and TryGetPrepared?

Reply:
Sure.